### PR TITLE
OSDOCS-9227 ALBO async release note update

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s): current

Issue: https://issues.redhat.com/browse/OSDOCS-9227

Link to docs preview: https://70999--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jmanthei** [comment]
@kowen-rh /label peer-review-needed

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc [hunk 1]
```diff
@@ -13,6 +13,11 @@ These release notes track the development of the AWS Load Balancer Operator in {
 
 For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
 
+[id="aws-load-balancer-operator-release-notes-1.1.z"]
+== AWS Load Balancer Operator 1.1.z
```
Comment (kowen-rh) [inline networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:17]: Do we know the version number being released? If so, we need the actual version number here and in the advisory notice. I see that it states "1.1.z" in the erratum, but I sincerely doubt that that's the actual version number.

@alebedev87 What version is this?

**Comment by kowen-rh** [comment]
The content looks good to me, but I'm not sure about the version number "1.1.z."

/remove-label peer-review-in-progress
/label peer-review-done

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc (hunk 1 above)
Comment (jmanthei) [reply networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:17]: @ShudiLi can you answer?

**Comment by jmanthei** [comment]
@ShudiLi for QE check please

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc (hunk 1 above)
Comment (ShudiLi) [reply networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:17]: I think it should be 1.1.1.
[aws-load-balancer-operator-bundle-container-1.1.1-4](https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2864555)

**Comment by ShudiLi** [comment]
/lgtm
thanks

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc (hunk 1 above)
Comment (alebedev87) [reply networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:17]: Yes, we shipped `1.0.1` and `1.1.1` versions for the rapid reset CVE fix on 30th of January.

**Comment by alebedev87** [comment]
/hold

Can be unhold once the version is updated to `1.1.1`.

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc [hunk 2]
```diff
@@ -13,6 +13,11 @@ These release notes track the development of the AWS Load Balancer Operator in {
 
 For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
 
+[id="aws-load-balancer-operator-release-notes-1.1.1"]
+== AWS Load Balancer Operator 1.1.1
+
+The following advisory is available for the AWS Load Balancer Operator version 1.1.z: link:https://access.redhat.com/errata/RHEA-2024:0555[RHEA-2024:0555 Release of AWS Load Balancer Operator 1.1.z on OperatorHub]
```
Comment (kowen-rh) [inline networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:19]: This line still has version 1.1.z.

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc [hunk 3]
```diff
@@ -13,6 +13,11 @@ These release notes track the development of the AWS Load Balancer Operator in {
 
 For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
 
+[id="aws-load-balancer-operator-release-notes-1.1.1"]
+== AWS Load Balancer Operator 1.1.1
+
+The following advisory is available for the AWS Load Balancer Operator version 1.1.1: link:https://access.redhat.com/errata/RHEA-2024:0555[RHEA-2024:0555 Release of AWS Load Balancer Operator 1.1.z on OperatorHub]
```
Comment (kowen-rh) [inline networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:19]: I just noticed that the link to the advisory should be a bullet point to be consistent with the previous release notes.

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc [hunk 4]
```diff
@@ -45,6 +50,11 @@ remote error: tls: protocol version not supported
 +
 Now, the AWS Load Balancer Controller accepts TLS 1.2 as the minimum TLS version, resolving this issue. (link:https://issues.redhat.com/browse/OCPBUGS-14846[*OCPBUGS-14846*])
 
+[id="aws-load-balancer-operator-release-notes-1.0.1"]
+== AWS Load Balancer Operator 1.0.1
+
+The following advisory is available for the AWS Load Balancer Operator 1.0.1 on OperatorHub. link:https://access.redhat.com/errata/RHEA-2024:0556[Release of AWS Load Balancer Operator 1.0.1 on OperatorHub]
```
Comment (kowen-rh) [inline networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:56]: Same here, the link to the erratum should be a bullet point. :slightly_smiling_face: 

For consistency, we should also include the word "version" before the version number. There's also no need to mention OperatorHub in the text before the link. Example:

```adoc
The following advisory is available for the AWS Load Balancer Operator version 1.0.1:

* link:https://access.redhat.com/errata/RHEA-2024:0556[Release of AWS Load Balancer Operator 1.0.1 on OperatorHub]
```

**Review by kowen-rh — CHANGES_REQUESTED** [review]
Minor changes are required, but I think this PR is ready for merge review once these few things are taken care of. Thanks for working on this, @jmanthei!

File: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc [hunk 5]
```diff
@@ -45,6 +52,13 @@ remote error: tls: protocol version not supported
 +
 Now, the AWS Load Balancer Controller accepts TLS 1.2 as the minimum TLS version, resolving this issue. (link:https://issues.redhat.com/browse/OCPBUGS-14846[*OCPBUGS-14846*])
 
+[id="aws-load-balancer-operator-release-notes-1.0.1"]
+== AWS Load Balancer Operator 1.0.1
+
+The following advisory is available for the AWS Load Balancer Operator version 1.0.1.
```
Comment (ousleyp) [inline networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc:58]: This line should end in a `:` to be consistent with the other RNs

**Review by ousleyp — COMMENTED** [review]
Just one super tiny nit. Thanks, Jessi!

**Comment by ousleyp** [comment]
LGTM; merging!

**Pull request merged**